R/tm_missing_data.R

```diff
@@ -123,7 +123,15 @@ ui_page_missing_data <- function(id, datasets, pre_output = NULL, post_output =
             datanames,
             function(x) {
               conditionalPanel(
-                sprintf("$(\"#%s > li.active\").text().trim() == \"%s\"", ns("dataname_tab"), x),
+                # supporting the bs3 and higher version at the same time
+                sprintf(
+                  paste0(
+                    "$(\"#%1$s > li.active\").text().trim() == '%2$s' || ",
+                    "$(\"#%1$s > li a.active\").text().trim() == '%2$s'"
+                  ),
```

Comment:
I wonder if we can refactor this out into a function as it is used in a few different places

Reply:
is_tab_active_js was created